daemons/pid/simple.py

```diff
@@ -32,12 +33,31 @@ def pid(self):
 
                 try:
 
-                    return int(pidfile.read().strip())
+                    pid = int(pidfile.read().strip())
 
                 except ValueError:
 
                     return None
 
+		try:
+
+		    os.kill(pid, 0)
+
+		except OSError as e:
+
+		    if e.errno == errno.EPERM:
+
+			return pid
```

Comment:
What is the use case for having the manager return a valid pid when the current process doesn't have the permissions needed to manage the process identified by the pid?